opentelemetry-api/setup.cfg

```diff
@@ -55,6 +55,8 @@ opentelemetry_context =
     contextvars_context = opentelemetry.context.contextvars_context:ContextVarsRuntimeContext
 opentelemetry_tracer_provider =
     default_tracer_provider = opentelemetry.trace:_DefaultTracerProvider
+opentelemetry_meter_provider =
+    default_meter_provider = opentelemetry._metrics:_DefaultMeterProvider
```

Comment:
I know we have done this before but `DefaultX` is not a descriptive name.

Reply:
Is there an alternative?

Reply:
NoOpTracerProvider, the problem with Default is that it is not an intrinsic characteristic of the class. Also, I'm not sure if it makes sense to have a private class exposed in a public entry point.

Reply:
This is the case here: https://github.com/open-telemetry/opentelemetry-python/pull/2354/files#diff-b0d8c5b726a14ec18c737b7728caa6d928cd13f910cc83ad47463c24989b831aR61

I like the idea of NoOpTracerProvider... is this captured anywhere?

Reply:
No, I can open an issue, but looks like a breaking change :shrug:

Reply:
Can always just be an alias with a deprecation message.